cohortextractor/tpp_backend.py

```diff
@@ -2805,6 +2805,35 @@ def patients_with_value_from_file(
     def patients_which_exist_in_file(self, f_path):
         return self.patients_with_value_from_file(f_path)
 
+    def patients_with_an_isaric_record(
+        self,
+        returning,
+        between=None,
+        date_filter_column=None,
+        return_expectations=None,
+    ):
+
+        queries = []
+        table = "ISARIC_Patient_Data_TopLine"
+
+        if between and date_filter_column:
+            date_condition, _ = self.get_date_condition(
```

Comment:
You need the second argument from this: it's a join condition that tells you what other tables need to be joined to if the date conditions are derived from other queries, rather than being static values.

Reply:
fixed